### ISSUE TITLE
Collection ABCs not used on Python 2.7

### ISSUE THREAD

**Issue by ctoth** [cdgriffith/Box#34]
On Python 2.7 there is no collections.abc module, instead the collection base types are available directly in the collections module
The current code says:
```
try:
    from collections.abc import Mapping, Iterable
except ImportError:
    Mapping = dict
    Iterable = (tuple, list)
```

I recommend putting from collections import Iterable, Mapping as the except case, possibly still including the fallback for even earlier Pythons but possibly not as this totally changes how the library works rather unexpectedly. At the very least, a warning should happen when this fallback mode is used.

**Comment by cdgriffith** [comment]
Great catch. I will update it to change it to `from collections import Iterable, Mapping` as suggested. 
There are other new factors in 3.x that break anything below 2.7 anyways.

**Comment by cdgriffith** [comment]
Fixed in 3.1 https://github.com/cdgriffith/Box/commit/252e2f574213708cbbcd42bdec442434384127fe
